metabase/metabase
# Fix pie invalid other percentages

Closes https://github.com/metabase/metabase/issues/49061

### Description

Fixes "Other" slice percentage calculation.

### How to verify

<img width="1406" alt="Screenshot 2024-10-23 at 8 50 56 PM" src="https://github.com/user-attachments/assets/6562eccb-0d65-4cfe-8be1-163a601b8c60">

- Create the question above. Inner ring = created at, outer ring = total
- Set "Minimum slice percentage" to 20
- Ensure "Other" percentages are computed correctly

### Demo

Before
<img width="584" alt="Screenshot 2024-10-23 at 8 50 03 PM" src="https://github.com/user-attachments/assets/ecdb8adb-e282-4e34-b631-9fe8bec098fc">

The most right "Other" slice percentage is label is incorrectly 27% but should be 77%

After
<img width="689" alt="Screenshot 2024-10-23 at 8 49 29 PM" src="https://github.com/user-attachments/assets/f6d7a1c6-9d62-406a-9f57-931e9b60d153">

### Checklist
